app/components/forms/order-form.js

```diff
@@ -0,0 +1,110 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    var validationRules = {};
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        firstname: {
+          for(var i = 1; i <= 3; i++){
```

Comment:
@niranjan94 I know it's incorrect but can you please suggest where to place the loop and the right code to solve this. Thanks.